python_scripts/check_multiqc.py

```diff
@@ -22,6 +22,10 @@
 from cpg_utils.config import get_config
 
 
+logging.basicConfig()
+logging.getLogger().setLevel(logging.DEBUG)
```

Comment:
You can combine these statements:

```py
logging.basicConfig(level=logging.DEBUG)
```

Do you think it's worth instantiating a logger here instead of using the default one? Often, `logger = logging.getLogger(__name__)` or similar is used at the top level (see https://docs.python.org/3/library/logging.html#logger-objects).

Reply:
> You can combine these statements:

Unfortunately, this is not an equivalent statement.`logging.basicConfig(level=logging.DEBUG)` sets the global level, and doesn't alter the root logger instance that would be instantiated by this point. I know that it's super confusing. See also https://stackoverflow.com/a/38537983

Reply:
> Do you think it's worth instantiating a logger here instead of using the default one?

I think using the root logger is just simpler, since I don't need multiple loggers, and also it's a small isolated script that is always run within a Batch job.